File: SupplyHub.Server/Dtos/Product/FetchProductResponseDto.cs
namespace Dtos.Product;

public class FetchProductResponseDto
{
	public required string ProductName { get; set; }
	public string? ProductType { get; set; }
	public byte[]? Thumbnail { get; set; }
	public byte[][]? Images { get; set; }
	public int? StockAvailable { get; set; }
	public required decimal Price { get; set; }
	public string? Unit { get; set; }
	public string? TimeFrame { get; set; }
	public string? Description { get; set; }
	public string[]? FaqQuestions { get; set; }
	public string[]? FaqAnswers { get; set; }
}

File: supplyhub.client/src/Dtos/Product/FetchProductResponseDto.ts
export interface FetchProductResponseDto {
    productID: number;
    productName: string;
    productType?: string;
    thumbnail?: string;
    images?: string[];
    stockAvailable?: number;
    price: number;
    unit?: string;
    faqQuestions?: string[]; 
    faqAnswers?: string[];   
    timeFrame?: string;
    description?: string;
    isActive?: boolean;
    dateAdded?: string;
}
